step.sh

```diff
@@ -213,7 +213,7 @@ if [ -n "${firebase_token}" ] ; then
 fi
 
 if [ -n "${release_notes}" ] ; then
-    submit_cmd="$submit_cmd --release-notes \"$(escape "$release_notes")\""
+    submit_cmd="$submit_cmd --release-notes=\"$(escape "$release_notes")\""
```

Comment:
This seems not be right. In the [docs](https://firebase.google.com/docs/app-distribution/android/distribute-cli) they have:

```
--release-notes "Text of release notes"
```

![image](https://github.com/guness/bitrise-step-firebase-app-distribution/assets/4848662/d27c60c6-3540-421a-9045-fbcbca7b0b7d)